integration-tests/bats/column_tags.bats

```diff
@@ -353,14 +353,6 @@ DELIM
     [ $status -eq 0 ]
     [[ $output =~ "| target | col1   | 14649 |" ]] || false
```

Comment:
Are these `tags` calls still relevant?

Reply:
oooh... good point. they aren't relevant to this test anymore. I'll yank 'em out!

Reply:
Maybe make a new test of `dolt schema tags` if it's not tested anywhere else?